tests/integration/test_user_endpoints.py

```diff
@@ -464,6 +464,65 @@ def test_user_follows_and_unfollows_user(self):
         # Unfollow these 2 users
         self.spotify.user_unfollow_users(users)
         self.assertFalse(any(self.spotify.current_user_following_users(users)))
+    
+    def test_album(self):
```

Comment:
this is already present in https://github.com/spotipy-dev/spotipy/blob/5201f58247f863dbe5eb8048684fa2f9e8e0a849/tests/integration/non_user_endpoints/test.py#L101